Makefile

```diff
@@ -120,3 +120,27 @@ build:
 docker:
 	@printf "  \033[38;5;154mDOCKER\033[0m  \033[38;5;176mBuilding production docker image\033[0m\n"
 	@docker build -t newdle .
+
+
+################ Docker-compose commands ################
+
+env:
```

Comment:
I would prefix all the makefile targets related to docker with `docker`...

Do we need anything except these btw?

- `make docker-run`: `docker-compose up` (without `-d`, this removes the need for the `logs` command)
- `make docker-clean`: `docker-compose down` and whatever else is needed to clean up the docker things
- `make docker-shell`: same as your `shell-env`

I've seen some packages (iirc mattermost does that for example) that run the docker stuff in the backgrond, but I think it makes it *very* easy to miss important error/warning output. I'm not sure how well foregrounded docker-compose with multiple container works. But if the output is readable it should be fine...

Reply:
I usually do it this way, but I usually use the compose environment for development, so maybe not all is useful indeed.  I think you suggestion will be just fine, I'll update it. Regarding docker-compose in the foreground I never had any issues.